### PR TITLE
COG-1199 Store events to Postgress

### PR DESCRIPTION
**Pull request by AlexanderSerbin**
### JIRA Issue Link
https://osi-cwds.atlassian.net/browse/COG-1199

### Technical Description
Stored audit events in Postgres database

### Tests
- [x] I have included unit tests 
- [x] I have included integration tests 
- [ ] I have included performance tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

### Types of changes
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Technical debt
!!!Correct rollback behavior for audit events is not covered

### CODE REVIEW

File: src/main/java/gov/ca/cwds/idm/service/AuditEventService.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    return enrichAuditEventWithId(new UserRegistrationResentEvent(user));
+  }
+
+  public UserCreatedEvent createUserCreatedEvent(User user) {
+    return enrichAuditEventWithId(new UserCreatedEvent(user));
+  }
+
+  private <T extends AuditEvent<S>, S extends Serializable> T enrichAuditEventWithId(
+      T auditEvent) {
+    auditEvent.setId(UUID.randomUUID().toString());
```
Comment (sorych) [inline src/main/java/gov/ca/cwds/idm/service/AuditEventService.java]: may this be moved to the constructor of a base class?

Comment (sorych) [inline src/main/java/gov/ca/cwds/idm/service/AuditEventService.java]: may id be a constructor parameter for an event?

Comment (AlexanderSerbin) [reply src/main/java/gov/ca/cwds/idm/service/AuditEventService.java]: Done

**Comment by AVandana-OSI** [comment]
retest this please

**Pull request merged**